src/extension.ts

```diff
@@ -1,39 +1,79 @@
 import * as vscode from 'vscode';
+import * as lang from './languages';
 
-import { search } from './utils/search';
-import { matchSearchPhrase } from './utils/matchSearchPhrase';
+try { process.removeAllListeners('warning'); } catch (e) { /* browser, probably. */ }
+
+const DEBUG = false;
+
+let run : Boolean = false
+
+/**
+ * Sends an HTTP request to the huggingface bloom API and returns the processed response.
+ * @param {String} text The text for the AI to compute.
+ * @param {Number} seed 
+ * @param {Boolean} early_stopping 
+ * @param {Number} length_penalty 
+ * @param {Number} max_new_tokens 
+ * @param {Boolean} do_sample 
+ * @param {Number} top_p 
+ * @returns {String} The text returned by the AI.
+ */
+const bloom = async (text : string, seed = 31, early_stopping = false, length_penalty = 0, max_new_tokens = 20, do_sample = false, top_p = 0.9) => {
+    const response = await fetch("https://api-inference.huggingface.co/models/bigscience/bloom", {
+        "headers": {
+            'Authorization': 'Bearer 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'
```

Comment:
Take that token out! That's a huge security risk.

Reply:
The token only has a READ permission, So it could only SEND requests.